src/components/TodoList.jsx

```diff
@@ -1,66 +1,206 @@
 import React from 'react';
 import TodoPoint from './TodoPoint';
+import { formatDate, filterTasks, sortingOptions } from './utils';
 
 const todoStyle = {
     display: "inline-block"
 }
 
-
-
 class TodoListXP extends React.Component {
     constructor(props) {
         super(props);
-
         this.state = {
+            // Array of todo elements as objects { name of task, 
+            // date when task added, id of task, color of task, 
+            // text decoration style of task }
             tasks: [
-                "do one thing",
-                "another one",
-                "last one to do"
+                {
+                    taskName: "do one thing", date: new Date(),
+                    id: 0, color: "", textDecoration: ""
```

Comment:
Please don't create objects/hashmaps like this, it's not readable. Consider this instead, one property per line:

![image](https://user-images.githubusercontent.com/3951613/91298235-54b6a780-e7a8-11ea-9c95-dadb236c5129.png)

Later on, when you hang out with cool kids from Haskell gang, you will also see comma-first notation, but it will be rejected by js formatter by default if you try using it in js:

![image](https://user-images.githubusercontent.com/3951613/91298393-a19a7e00-e7a8-11ea-8324-dd4a40d057a6.png)